### ISSUE TITLE
fading letter

### ISSUE THREAD

**Issue by JeePopuwu** [fast4x/RiMusic#4934]
### Description

add a fade animation as shown in the video:

### Suggested Solution

https://github.com/user-attachments/assets/730d95d7-e937-42e0-999b-c6cbcc40bd93

**Comment by Homelander5i** [comment]
Yes you should add this animation :) and why you don't add achievements ? It will be cool 😎 and a booting animation!
